Signal/Images.xcassets/profile_avatar_default.imageset/Contents.json

```diff
@@ -0,0 +1,23 @@
+{
+  "images" : [
+    {
+      "idiom" : "universal",
+      "filename" : "profile_avatar_default@1x.png",
+      "scale" : "1x"
+    },
+    {
+      "idiom" : "universal",
+      "filename" : "profile_avatar_default@2x.png",
+      "scale" : "2x"
+    },
+    {
+      "idiom" : "universal",
+      "filename" : "profile_avatar_default@3x.png",
+      "scale" : "3x"
+    }
+  ],
+  "info" : {
+    "version" : 1,
+    "author" : "xcode"
+  }
```

Comment:
![screen shot 2017-08-01 at 11 46 49 am](https://user-images.githubusercontent.com/625803/28834013-218a02ac-76af-11e7-961e-89b9aee330c7.png)

Note: This overall view design is just provisional.